apps/ui/src/composables/useSpaces.ts

```diff
@@ -133,33 +134,43 @@ export function useSpaces() {
   }
 
   async function fetch(filter?: SpacesFilter) {
-    if (loading.value || loaded.value) return;
     loading.value = true;
-
     await _fetchSpaces(true, filter);
 
     loaded.value = true;
     loading.value = false;
   }
 
   async function fetchMore(filter?: SpacesFilter) {
-    if (loading.value || !loaded.value) return;
+    if (loading.value || !loaded.value || loadingMore.value) return;
     loadingMore.value = true;
 
     await _fetchSpaces(false, filter);
 
     loadingMore.value = false;
   }
 
+  async function handleProtocolChange(newProtocol: ExplorePageProtocol) {
+    protocol.value = newProtocol;
+    await fetch();
+  }
```

Comment:
Not sure it makes sense to export two things that can be used to change protocol.

Now we can do it via `protocol` which will trigger this function, or via `handleProtocolChange`, which will trigger `protocol` ref change. I feel like just exposing `protocol` is enough as it seems confusing (and we don't use `handleProtocolChange` anyway, so we could just watch protocol and fetch on change.

Reply:
True that thank you, I was using this function in the `Explore` page before but failed to remove it later 😄